app/assets/stylesheets/common/form-kit/_field.scss

```
@@ -35,6 +35,10 @@
     }
   }
 
+  .form-kit__container-description {
+    margin: 0;
```

lets decrease the font-size a bit
```suggestion
    margin: 0;
    font-size: var(--font-down-1-rem);
```